---
source: crates/ruff_linter/src/rules/flake8_pytest_style/mod.rs
---
error[PT010]: PT010.py:5:10: set the expected exception in `pytest.raises()`

  |
4 | def test_ok():
5 |     with pytest.raises():
  |          ^^^^^^^^^^^^^
6 |         pass
  |
